### ISSUE TITLE
VPN cannot be turned on if Network Settings ->“Local network access” options is checked

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla-mobile/mozilla-vpn-client#190]
#### Affected versions:
- 2;

#### Affected Platforms:
- Android 10;

#### Prerequisites:
- Have the VPN app installed;
- Have a FxA account with active subscription for Mozilla VPN;

#### Steps to reproduce:
1. Open the VPN app; (without turning it on)
2. Go to “Settings” ;
3. Go to “Network settings”;
4. Check the ”Local network access” option;
5. Go back in first screen and turn on the VPN;

#### Expected result:
- The VPN app is turned ON.

#### Actual result:
- The VPN app cannot be turned ON.

**Comment by ValentinaPC** [comment]
This is happening again on Android and Linux, as mentioned in the first comment.
Not reproducing on the latest cross-platform builds from MacOS and iOS.
![ezgif com-video-to-gif(2)](https://user-images.githubusercontent.com/15685960/99389861-5a6fe500-28e0-11eb-8102-fbb8d0033677.gif)

**Comment by ValentinaPC** [comment]
Verified this on the last Android build (the one from 11/18) and this is partially working.
VPN **can** be turned **on** while the "wi-fi" option is checked, and **cannot** be turned **on** while the "Local network access" options is checked.

**Comment by bakulf** [comment]
@strseb can you please take a look?

**Comment by bakulf** [comment]
We are about to postpone the captive portal feature.

**Comment by bakulf** [comment]
@strseb same issue happens when we want to allow local network addresses.

**Comment by ValentinaPC** [comment]
- As the Wi-Fi option (captive portal feature) was removed, I've updated the issue and its title.
- As the "Local network access" option was removed from Linux -> I've removed the "linux" label.

Verified as fixed on the last cross-plat build while using Android 10.
Selecting different locations and checking the "Local network access" option, does not block the VPN activation.
See below video.
![ezgif com-video-to-gif(4)](https://user-images.githubusercontent.com/15685960/100209822-ff0eaa00-2f12-11eb-809e-447a5dcd8746.gif)
